### PR TITLE
Player kw

### PR DESCRIPTION
**Pull request by fading-paths-dev**
Added a new player for the tourney

### CODE REVIEW

File: oxo_tourney/tests/player/test_restful_player.py [hunk 1]
```diff
@@ -1,36 +1,23 @@
...
         board_mock = MagicMock()
-        board_mock.size = 3
+        board_mock.size = 2
         board_mock.__str__.return_value = "...\n...\n..."
 
         move = player.get_move(board_mock, "X")
 
         assert_that(len(move), equal_to(2))
-
-    def test_get_move_diplays_warning_when_grid_size_not_3(self):
```
Comment (grahame-student) [inline oxo_tourney/tests/player/test_restful_player.py]: Test case reverted

**Review by grahame-student — CHANGES_REQUESTED** [review]
Trying out the code review feature

File: requirements.txt [hunk 2]
```diff
@@ -2,4 +2,4 @@ pytest==7.1.2
 pytest-cov==3.0.0
 pyhamcrest==2.0.3
 mock==4.0.3
-requests==2.28.1
\ No newline at end of file
+requests==2.28.0
```
[the remark this answers is not in the document]
Comment (grahame-student) [reply requirements.txt:5]: @fading-paths-dev the requests library needs to be set to the correct version (check main branch if you're unsure what that should be)

[the remark this answers is not in the document]
Comment (fading-paths-dev) [reply requirements.txt]: @grahame-student done

**Review by grahame-student — APPROVED** [review]

**Pull request merged**